install-dev/init.php

```diff
@@ -30,6 +30,12 @@
     die('You need at least PHP 5.4 to run PrestaShop. Your current PHP version is '.PHP_VERSION);
 }
 
+/* in dev mode - check if composer was executed */
+if (is_dir(_PS_ROOT_DIR_.DIRECTORY_SEPARATOR.'admin-dev') && (!is_dir(_PS_ROOT_DIR_.DIRECTORY_SEPARATOR.'vendor') ||
```

Comment:
Yeah but, I could have my folder `admin-doge` with a git version, the warn would never appear. :disappointed: 
The important point here is the vendor folder and its autoload file inside it. No need to test the admin folder in my opinion.